envoy/service/auth/v2alpha/attribute_context.proto

```diff
@@ -0,0 +1,133 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.service.auth.v2alpha;
```

Comment:
These are just copies from the existing protos? How come the existing ones aren't being deleted (i.e. this isn't a move)?

Reply:
Yes, I just copied them. I wasn't 100% sure if I should delete the old ones.